dd-java-agent/instrumentation/couchbase-2.0/src/main/java/datadog/trace/instrumentation/couchbase/client/CouchbaseBucketInstrumentation.java

```diff
@@ -118,8 +119,8 @@ public void call() {
           GlobalTracer.get()
               .buildSpan("couchbase.call")
               .withTag(DDTags.RESOURCE_NAME, resourceName)
-              .withTag("bucket", bucket)
               .start();
+      Tags.DB_INSTANCE.set(span, bucket);
```

Comment:
any reason not to just update `"bucket"` with the db instance key accessor?

Reply:
I'm not really following